transforms/universal/hap/python/README.md

```diff
@@ -48,6 +56,20 @@ python hap_local_python.py
 
 You will obtain the output file `test1.parquet` in the output directory.
 
+### Code example
+[notebook](../hap_python.ipynb)
```

Comment:
The link to the notebook is broken. The correct link is: ./hap_python.ipynb, because the notebook is in the python directory.